### ISSUE TITLE
Use some fixed colors generate the rest

### ISSUE THREAD

**Issue by stitam** [jolars/qualpalr#5]
Hi, many thanks for developing this package! I have an application where I have some fixed colors and want to find optimal colors for the rest of the character levels using qualpalr. If I need "n" colors altogether and "k" of them are fixed I can of course generate "n-k" new colors and combine the two sets but then some of them might overlap accidentally. I was wondering if there is a way to handle this with qualpal? If not, would this be difficult to implement? Thanks.

**Comment by jolars** [comment]
Good idea. I think that would definitely be possible. Right now I don't have much time to devote to this, however, but I will try to find some time in the coming weeks.

**Comment by jolars** [comment]
So, two years later, I finally got around to implement this, @stitam :)

**Comment by stitam** [comment]
No worries, thanks @jolars for implementing the feature!
